### PR TITLE
Fix GIFs not sharing

### PR DESCRIPTION
**Pull request by zfox23**
Whoops. After a previous PR of mine, GIFs didn't share.

*Test Plan:*
1. Click Snap
2. Wait for review dialog to appear and GIF to process
3. Select the checkbox to the left of the GIF
4. Click Share in Feed
5. Verify that the GIF shows up in the feed

### CODE REVIEW

File: scripts/system/snapshot.js [hunk 1]
```diff
@@ -239,6 +239,7 @@ function processingGifStarted(pathStillSnapshot) {
         {
             containsGif: true,
             processingGif: true,
+            loadingGifPath: Script.resolvePath(Script.resourcesPath() + 'icons/loadingDark.gif'),
```
Comment (davidkelly) [inline scripts/system/snapshot.js]: 👍

**Comment by hifi-gustavo** [comment]
Build console output for commit: 85e5173abe95346bce6b4264376d8231fb7a9ca8 available at: https://gist.github.com/902908a8f91495917c87b1aa9a83e3d2<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10224-85e5173.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10224-85e5173.dmg

**Comment by claudioshg** [comment]
testing

**Comment by hifi-gustavo** [comment]
Build console output for commit: b410399a16eb73da9c49b64f7ff16c208d04d318 available at: https://gist.github.com/902908a8f91495917c87b1aa9a83e3d2<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10224-b410399.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10224-b410399.dmg

**Comment by claudioshg** [comment]
testing

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10224-b410399.exe

TESTED:
- Toolbar in Desktop Mode: FAIL
- Toolbar in HMD Mode: PASS
- Tablet in Desktop Mode: PASS
- Tablet in HMD Mode: PASS

FAILS:
Toolbar in Desktop Mode: 
- Click Snap: Success
- Wait for review dialog to appear and GIF to process: Success
- Select the checkbox to the left of the GIF: Success
- Click Share in Feed: Success
- Verify that the GIF shows up in the feed: FAIL: Clicking on the "Share in Feed" button does nothing. 
A window/dialog doesn't open, or anything happen.
The "Don't Share" button and "Settings" does nothing as well.

NOTES:
- Tested while logged in and logged out.
- When I used the Tablet in Desktop Mode, I clicked on "Share in Feed" and I was promted to log in. 
I clicked on Cancel and went back to the dialog where I'm shown the GIF and picture.
However, the GIF and picture were missing.

**Comment by zfox23** [comment]
@claudioshg Thanks for the QA. Can you please verify that the failing behavior is present in the latest [dev-download](https://highfidelity.io/dev-download)? I believe it is, since I didn't change anything about the "Share in Feed" button, and this failure has been noted before.

If this behavior is present in the latest dev-download, and this behavior is the only thing causing the fail, then this is a QA Pass.

**Comment by claudioshg** [comment]
@zfox23 Checking the latest dev-download

**Comment by claudioshg** [comment]
@zfox23 The behavior is present in the latest dev-download build (6331). Cool! This PR is a QA Pass.

**Pull request merged**